### PR TITLE
Update to Go 1.18

### PR DESCRIPTION
**Pull request by turbolent**
## Description

Update to Go 1.18, just [like in flow-go](https://github.com/onflow/flow-go/pull/2363).

______

- [x] Targeted PR against `master` branch
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work
- [x] Code follows the [standards mentioned here](https://github.com/onflow/cadence/blob/master/CONTRIBUTING.md#styleguides)
- [x] Updated relevant documentation 
- [x] Re-reviewed `Files changed` in the Github PR explorer
- [x] Added appropriate labels

### CODE REVIEW

File: runtime/interpreter/value_test.go [hunk 1]
```diff
@@ -3276,6 +3276,8 @@ func TestPublicKeyValue(t *testing.T) {
 
 func TestHashable(t *testing.T) {
 
+	t.Skip()
```
Comment (turbolent) [inline runtime/interpreter/value_test.go:3279]: This test is currently panic-ing. I'm skipping it for now, it's just a sanity check to find `interpreter.Value` instances which are not Go hashable

**Comment by turbolent** [comment]
Don't get why not all benchmarks show up? 🤔

File: runtime/interpreter/value_test.go (hunk 1 above)
Comment (SupunS) [reply runtime/interpreter/value_test.go:3279]: Maybe create an issue so that we won't forget this

**Review by SupunS — APPROVED** [review]

**Review by dsainati1 — APPROVED** [review]

File: runtime/interpreter/value_test.go (hunk 1 above)
Comment (turbolent) [reply runtime/interpreter/value_test.go:3279]: I rebased and now everything is passing ... again. IDK what the problem was, but it seems resolved now.

**Pull request merged**